torchci/clickhouse_queries/last_branch_push/query.sql

```diff
@@ -0,0 +1,15 @@
+-- Elapsed seconds since the last commit was pushed to pytorch/pytorch's main branch
+
+select
+    DATE_DIFF('second', head_commit.timestamp, CURRENT_TIMESTAMP()) as push_seconds_ago
+from
+    push
+where
+    push.ref = { branch : String }
+    AND push.repository.owner.name = 'pytorch'
+    AND push.repository.name = 'pytorch'
+    AND push.head_commit is not null
```

Comment:
Nit: I learn from @clee2000 that we need to pay close attention to NULL in ClickHouse.  A column need to be defined as `nullable` to contain null value I think.  From what I see the `head_commit` is defined as a tuple of:

```
`head_commit` Tuple(added Array(String), author Tuple(email String, name String, username String), committer Tuple(email String, name String, username String), `distinct` Bool, id String, message String, modified Array(String), removed Array(String), timestamp DateTime64(9), tree_id String, url String),
```

So the field couldn't be null and I think you can remove the redundant `AND push.head_commit is not null` if you prefer to simplify the query a tiny bit.